packages/ngircd/fix-hardcoded-path.patch

```diff
@@ -0,0 +1,64 @@
+--- ngircd-26/autogen.sh	2021-11-01 11:27:20.660709063 +0700
++++ ngircd-26.mod/autogen.sh	2021-11-01 11:54:22.496709160 +0700
```

Comment:
Why do you need to change the autogen script? You do not seem to be running it in the build

Reply:
Autogen scripts and build script lines using tools shouldn't be patched.

Reply:
Autogen script is locating to /bin instead of Termux prefix path. The /bin path is used for detecting autotool and automake.

Reply:
But you are not running autogen.sh in the build, so why would this be a problem?

Reply:
Uh oh, i see. Maybe i can remove this patch